src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const EVERY_DAY_COST = 40;
+  const DISCOUNT_AFTER_3_DAYS = 20;
+  const DISCOUNT_AFTER_7_DAYS = 50;
+
+  if (days <= 2) {
+    return days * EVERY_DAY_COST;
+  }
+
+  if (days <= 6) {
+    return days * EVERY_DAY_COST - DISCOUNT_AFTER_3_DAYS;
+  }
+
+  if (days >= 7) {
```

Comment:
Instead of using `if (days >= 7)`, you can simply use an `else` block here, as the previous conditions already ensure that `days` is at least 7 if execution reaches this point.